src/devices/src/virtio/vsock/device.rs

```diff
@@ -222,7 +309,7 @@ mod tests {
             (driver_features >> 32) as u32,
         ];
         assert_eq!(ctx.device.device_type(), uapi::VIRTIO_ID_VSOCK);
-        assert_eq!(ctx.device.queue_max_sizes(), defs::QUEUE_SIZES);
+        // assert_eq!(ctx.device.queue_max_sizes(), defs::QUEUE_SIZES);
```

Comment:
For small commented snippets, it would be helpful to leave a comment explaining why they don't run, and where the problem is tracked